src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -12,19 +1,63 @@
+type Props = {
+  setOptionsValue: (event: string) => void;
+  setInputValue: (event: string) => void;
+  inputValue: string;
+};
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({
+  setOptionsValue,
+  setInputValue,
+  inputValue,
+}) => {
+  function handleOptionValue(event: React.ChangeEvent<HTMLInputElement>) {
+    setOptionsValue(event.target.value);
+  }
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+  function handleInputValue(event: React.ChangeEvent<HTMLSelectElement>) {
+    setInputValue(event.target.value);
```

Comment:
The event type for `handleInputValue` should be `React.ChangeEvent<HTMLInputElement>` instead of `React.ChangeEvent<HTMLSelectElement>`, as it handles changes for an `<input>` element.